src/styles/input-styles.js

```diff
@@ -24,6 +24,11 @@ export default css`
   background-color: var(--primary-color);
   color: var(--primary-btn-text-color);
 }
+.m-btn.primary:hover,
+.m-btn.primary:focus-visible {
+  color: var(--primary-color);
+  background-color: var(--primary-btn-text-color);
+}
```

Comment:
I think this actually breaks the experience:
![Peek 2025-05-12 16-08](https://github.com/user-attachments/assets/973a0773-608a-4056-a726-d9c7a4cc8c80)


I don't know what the correct fix is, it looks like there might be a mistake here in the underlying library not marking these as `active`.


My suggestion is going to be revert this for the PR, and then we can come up with a separate solution out of band so the rest of these changes can be merged in, unless you have an obvious solution that will fix this.

One idea that comes to mind is changing the border/text color like we do with the nav items, without changing the background. Thoughts?

Reply:
The hover state for the non-active buttons makes them look active (currently selected). The hover state for the selected buttons, currently, does not exist. The obvious quick thing to me was to reverse it, but if I were designing my own experience from scratch each button style (selected and normal) would have its own hover style that *did not overlap* with the active/selected style.